ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/RecoveryTest.java

```diff
@@ -666,125 +662,4 @@ public void shouldRecoverLogWithTerminateAfterDrop() {
     shouldRecover(commands);
   }
 
-  @Test
-  public void shouldCascade4Dot1Drop() {
-    commands.addAll(
-        ImmutableList.of(
-            new QueuedCommand(
-                new CommandId(Type.STREAM, "A", Action.CREATE),
-                new Command(
-                    "CREATE STREAM A (COLUMN STRING) "
-                        + "WITH (KAFKA_TOPIC='A', VALUE_FORMAT='JSON');",
-                    true,
-                    Collections.emptyMap(),
-                    null
-                ),
-                Optional.empty(),
-                0L
-            ),
-            new QueuedCommand(
-                new CommandId(Type.STREAM, "A", Action.CREATE),
-                new Command(
-                    "CREATE STREAM B AS SELECT * FROM A;",
-                    true,
-                    Collections.emptyMap(),
-                    null
-                ),
-                Optional.empty(),
-                1L
-            )
-        )
-    );
-    final KsqlServer server = new KsqlServer(commands);
-    server.recover();
-    assertThat(
-        server.ksqlEngine.getMetaStore().getAllDataSources().keySet(),
-        containsInAnyOrder(SourceName.of("A"), SourceName.of("B")));
-    commands.add(
-        new QueuedCommand(
-            new CommandId(Type.STREAM, "B", Action.DROP),
-            new Command("DROP STREAM B;", true, Collections.emptyMap(), null),
-            Optional.empty(),
-            2L
-        )
-    );
-    final KsqlServer recovered = new KsqlServer(commands);
-    recovered.recover();
-    assertThat(
-        recovered.ksqlEngine.getMetaStore().getAllDataSources().keySet(),
-        contains(SourceName.of("A")));
-  }
-
-  @Test
-  public void shouldUseOldQueryIdGenerationAndNewGeneration() {
```

Comment:
I think we could keep some version of this test, maybe rename to `shouldUseSpecificQueryIdGenerator` to make sure the offsets are being set correctly still.